### PR TITLE
Documentation: Update links/commands and make minor adjustments.

### PR DESCRIPTION
**Pull request by timflannagan1**

### CODE REVIEW

File: Documentation/dev/debugging.md [hunk 1]
```diff
@@ -21,10 +21,16 @@ kubectl get pods -n $METERING_NAMESPACE -l app=reporting-operator -o name | cut
...
 The following will open up an interactive presto-cli session where you can interactively query Presto. One thing to note is that this runs in the same container as Presto and launches an additional Java instance, meaning you may run into memory limits for the pod. If this occurs, you should increase the memory request & limits of the Presto pod.
 
+Interacting with presto-cli on macOS:
 ```
 kubectl get pods -n $METERING_NAMESPACE -l app=presto,presto=coordinator -o name | cut -d/ -f2 | xargs -o -I{} kubectl -n $METERING_NAMESPACE exec -it {} -- /usr/local/bin/presto-cli --server localhost:8080 --catalog hive --schema default --user root
 ```
 
+Interacting with presto-cli on Linux:
+```
+kubectl -n $METERING_NAMESPACE exec -it "$(kubectl get pods -l app=presto,presto=coordinator -o name | cut -d/ -f2)"  -- /usr/local/bin/presto-cli --server localhost:8080 --catalog hive --schema default --user root
```
Comment (chancez) [inline Documentation/dev/debugging.md]: We can just use the subprocess option for both mac/linux in all spots. No need to have a different command for each.

File: Documentation/dev/debugging.md [hunk 2]
```diff
@@ -65,10 +71,16 @@ presto:default>
 
 The following will open up an interactive beeline session where you can interactively query Hive. One thing to note is that this runs in the same container as Hive and launches an additional Java instance, meaning you may run into memory limits for the pod. If this occurs, you should increase the memory request & limits of the Hive pod.
 
+For interacting with the hive container on macOS:
 ```
 kubectl get pods -n $METERING_NAMESPACE -l app=hive,hive=server -o name | cut -d/ -f2 | xargs -o -I{} kubectl -n $METERING_NAMESPACE exec -it {} -c hiveserver2 -- beeline -u 'jdbc:hive2://127.0.0.1:10000/default;auth=noSasl'
 ```
 
+For interacting with the hive container on Linux:
```
Comment (chancez) [inline Documentation/dev/debugging.md]: We can use this for both mac/linux

File: Documentation/dev/debugging.md [hunk 3]
```diff
@@ -127,16 +139,16 @@ You can now open http://127.0.0.1:10002 in your browser window to view the Hive
...
 ```
 
-You can now open http://127.0.0.1:50070 in your browser window to view the HDFS web interface.
+You can now open http://127.0.0.1:9870 in your browser window to view the HDFS web interface.
 
 
 To the first datanode:
 
 ```
-kubectl -n $METERING_NAMESPACE port-forward hdfs-datanode-0 50075
```
Comment (chancez) [inline Documentation/dev/debugging.md]: Wow this hasn't been tested in a while I guess

**Comment by timflannagan1** [comment]
@chancez makes sense, updated.

File: Documentation/writing-custom-queries.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
 # Writing Custom Report Prometheus Queries and Report Generation Queries
 
-One of the main goals of Operator Metering is to be flexible, and extendible.
+One of the main goals of Operator Metering is to be flexible, and extendable.
 The way this has been done is to use Kubernetes Custom Resources as a way of letting users add to, or expand upon the built-in reports and metrics that the operator already has.
```
Comment (EmilyM1) [inline Documentation/writing-custom-queries.md]: How about extensible?

File: Documentation/dev/debugging.md (hunk 1 above)
Comment (chancez) [reply Documentation/dev/debugging.md]: The inner subshell is missing the ` -n $METERING_NAMESPACE` flag

**Pull request merged**